### PR TITLE
Emojification ✨

### PR DESCRIPTION
**Pull request by CadeXDyeDurham**
## Context 📙

It looks awesome and adds some enjoyment to looking at these blasted PR documents
> Also Added E2E cause it's useful

## Types of changes 🚀

- [ ] Bug fix (non-breaking change which fixes an issue) 🐛
- [x] New feature (non-breaking change which adds functionality) 🔥
- [ ] Breaking change (fix or feature that would cause existing functionality to change) ⚠️
- [ ] Other (Provide a brief description below) ❓

## Related Issue 🔗

[It looks good right?](https://github.com/dyedurham/gsf2-actions/pull/24)

## How Has This Been Tested? 📝

- [ ] Unit tests 🧪
- [ ] E2E tests 🛫🛬
- [x] Other (Provide a brief description below) ❓
- [x] You, you will test it 👁️ 👁️
- [x] This PR 📃

## Checklist ✅

- [x] I have updated the documentation if/where required. 👍
- [ ] I have added tests to cover my changes. 🛠️
- [x] All new and existing tests passed. 💯
- [x] No high or critical vulnerabilities detected. 🌈

How are these emojis for the checklist? The rainbow feels so satisfying for no vulnerabilities 😸

### CODE REVIEW

**Review by dinindu — APPROVED** [review]
Nice :rocket:

**Review by dinindu — APPROVED** [review]
Nice :rocket:

**Comment by rusty-1-shaun** [comment]
@CadeXDyeDurham I'm assuming the  🟢 🔶 🟥 emojis are there to indicate urgency, but the danger of having only emoji's are that the meaning can be miss interpreted. It's a great idea in trying to bring fun to a PR's.

Like art 🎨, it can be very subjective, so I'm not shooting your PR down 💥. I find that the use of too many emojis becomes distracting from the PR content (it starts to look very busy)  and would like to see them being used to draw attention to something that the audience should take note of. I'm not asking you to change anything but simply providing you with my subjective feedback 😊

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,37 +1,38 @@
-## Context
+## Context 📙
 
 <!--- Why is this change required? What problem does it solve? -->
 
-## Types of changes
+## Types of changes 🟢 🔶 🟥
```
Comment (rusty-1-shaun) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: To avoid any ambiguity please clarify what there emojis convey

**Review by rusty-1-shaun — DISMISSED** [review]

**Comment by CadeXDyeDurham** [comment]
> @CadeXDyeDurham I'm assuming the 🟢 🔶 🟥 shapes are there to indicate urgency, but the danger of having only emoji's are that the meaning can be miss interpreted. It's a great idea to bring fun to a PR's.
> 
> Like art 🎨, it can be very subjective, so I'm not shooting your PR down 💥. I find that the use of too many emojis becomes distracting from the PR content (it starts to look very busy) and would like to see them being used to draw attention to something that the audience should take note of. I'm not asking you to change anything but simply providing you with my subjective feedback 😊

I completely agree! Very well said, thank you for the valuable feedback. I was also dubious about what to use specifically for here, and I thought maybe 3 shapes were simply too many (distracting).

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (CadeXDyeDurham) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: They were to indicate the different types of changes:
- green = safe change (introduces no side-effects)
- yellow = partially safe change (might introduce minor side-effects but not major)
- red = breaking or unsafe change (will introduce major side-effects)

Comment (CadeXDyeDurham) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: how does 🚀 seem instead? I've updated the description to reflect it in the PR doc visualised

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,37 +1,38 @@
...
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
-- [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
-- [ ] No high or critical vulnerabilities detected.
\ No newline at end of file
+- [ ] My change requires a change to the documentation. ✍️
+- [ ] I have updated the documentation accordingly. 👍
```
[the remark this answers is not in the document]
Comment (CadeXDyeDurham) [reply .github/PULL_REQUEST_TEMPLATE.md:35]: fantastic idea!

**Review by rebecca-sell — APPROVED** [review]

**Review by emma-holt — APPROVED** [review]

**Pull request merged**